metacards/code.ts

```diff
@@ -0,0 +1,140 @@
+// This plugin will open a window to prompt the user to enter a number, and
```

Comment:
let's remove the commented code in lines 1 -2. do you have a short description to include instead?

Reply:
I added this description in the root directory's readme: 
> This plugin counts & organizes all of the votes (i.e. stamps) on stickies.